composer.json

```diff
@@ -24,14 +24,15 @@
         "symfony/event-dispatcher": "4.4.*|^5.0",
         "symfony/http-foundation": "4.4.*|^5.0",
         "symfony/http-kernel": "4.4.*|^5.0",
-        "symfony/options-resolver": "^5.0"
+        "symfony/options-resolver": "4.4.*|^5.0"
```

Comment:
Are you sure about this ? I think we put 5.0 because we use prototype options. 
I think the graphql annotation is always loaded even if not usable if the graphql package is not installed.

Reply:
@SofLesc No, i'm not sure, all other symfony lib have `4.4.*|^5.0` so i decided to  put the same but i can remove that

Reply:
I don't think this line works on 4.4 
https://github.com/BedrockStreaming/RateLimitBundle/blob/24e126b1978b5c3f7738a5c1e4531c7b3fe2244d/src/Annotation/GraphQLRateLimit.php#L25